SPECS/softhsm/softhsm.spec

```diff
@@ -1,29 +1,28 @@
-Vendor:         Microsoft Corporation
-Distribution:   Mariner
 #global prever rc1
 #global prerelease yes
```

Comment:
Please also remove these comments and clean-up the spec where these macros are used accordingly. No point to leave this lingering.